api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -549,3 +550,95 @@ def create_app_beneficiary() -> None:
         lastName=Fake.last_name(),
         needsToFillCulturalSurvey=False,
     )
+
+
+def create_venues_with_practical_info_graphical_edge_cases() -> None:
+    offerers_factories.VenueFactory(
+        name="Lieu avec un nom très long, qui atteint presque la limite de caractères en base de donnée et qui prend vraiment toute la place sur l'écran",
```

Comment:
[typo]

```suggestion
        name="Lieu avec un nom très long, qui atteint presque la limite de caractères en base de données et qui prend vraiment toute la place sur l'écran",
```

idem plus bas